### ISSUE TITLE
Bash command history scroll causes crash

### ISSUE THREAD

**Issue by blechinger** [Microsoft/WSL#2368]
* Windows build number: Microsoft Windows [Version 10.0.15063]

* What you're doing and what's happening:  Attempting to scroll through command history (buffer) using the keyboard's up or down arrows.

* What's wrong: Bash.exe crashes at end of buffer. If no buffer Bash crashes immediately (the instance seems to remain open in the background, however; the window freezes and then closes. Bash must be terminated manually via Task Manager. Launching bash again before manually terminating the application creates a new instance of Bash). The same behavior is seen when attempting tab completion (though less consistently).

* What should be happening instead: Bash should continue to run as normal?

* Strace of the failing command, if applicable: I followed the instructions from Jack Hammons provided [here](https://msdn.microsoft.com/en-us/commandline/wsl/troubleshooting). Four .elm files have been attached for reference as [lxcore_service.zip](https://github.com/Microsoft/BashOnWindows/files/1173447/lxcore_service.zip) (the article says there should be three but the .cmd scripts generated four).

I am running the latest (non Insider Preview) version of the application. I have already successfully uninstalled and reinstalled the application and see the same behavior. I am happy to assist with further troubleshooting/log gathering if it is helpful.

Thank you!

- Derek

**Comment by benhillis** [comment]
I'm not able to reproduce this, can you think of any other steps that would be required?  Do you have a custom .bashrc perhaps?

**Comment by blechinger** [comment]
Hey benhillis. Thanks for the response. :)

Pastebin of _cat .bashrc_ [here](https://pastebin.com/B5W2WULD). I don't recall modifying this.

I've got two Windows 10 boxes on the same release. I can't reproduce this on computer number 2. I always encounter this issue on computer number 1. That's when I went ahead and fully blew the install away and reinstalled. I came here when the issue persisted regardless.
